Add token examples of python (#25)
streamnative/pulsar-examples: cloud/node/README.md

# Overview

This document describes how to produce message to and consume message from a Pulsar cluster using [Apache pulsar-client-node](https://github.com/apache/pulsar-client-node).

# Prerequisites

Pulsar Node.js client library is based on the C++ client library. Follow the instructions for
[C++ library](https://pulsar.apache.org/docs/en/client-libraries-cpp/) for installing the binaries through
[RPM](https://pulsar.apache.org/docs/en/client-libraries-cpp/#rpm),
[Deb](https://pulsar.apache.org/docs/en/client-libraries-cpp/#deb) or
[Homebrew packages](https://pulsar.apache.org/docs/en/client-libraries-cpp/#macos).

> #### Note

> > you will need to install not only the pulsar-client library but also the pulsar-client-dev library.

Reviewer: ```suggestion
> You need to install both the pulsar-client library and the pulsar-client-dev library.
```

Also, this library works only in Node.js 10.x or later because it uses the
[node-addon-api](https://github.com/nodejs/node-addon-api) module to wrap the C++ library.

## Install 

> ### Please install pulsar-client in your project:

Reviewer: ```suggestion
### Install pulsar-client in your project
```
